docs/standard/analyzers/platform-compat-analyzer.md

```diff
@@ -0,0 +1,366 @@
+- Both attributes can be instantiated with or without version numbers as part of the platform name.
+  - Version numbers are in the format of `major.minor[.build[.revision]]`; `major.minor` is required and the `build` and `revision` parts are optional. For example, "Windows7.0" indicates Windows version 7.0, but "Windows" is interpreted as Windows 0.0.
+- Platform attributes can be applied to types, members (methods, fields, properties, and events) and assemblies.
+  - Attributes applied at the top level `target` affect all of its members and types.
+- The same `platform` attribute can be applied across the symbol tree for different symbols only with different versions. For example, an assembly has a `[SupportedOSPlatform("windows")]` attribute and there is a new type added with `Windows 10.0.17763.0` release; that type should be annotated with `[SupportedOSPlatform("window10.0.17763.0")]`.
+  - If `[SupportedOSPlatform("platformVersion")]` is applied more than twice for the referenced member with the same `platform`, only the minimum and maximum versions are considered by the analyzer.
+  - If `[UnsupportedOSPlatform("platformVersion")]` is applied more than twice for the referenced member with the same `platform`, only the two earliest versions are considered by the analyzer.
+
+For more information, see [examples of how the attributes work and what diagnostics they cause](#examples-of-how-the-attributes-work-and-what-diagnostics-they-produce).
+
+### Advanced scenarios for combining attributes
+
+- If a combination of `[SupportedOSPlatform]` and `[UnsupportedOSPlatform]` attributes are present, all attributes are grouped by OS platform identifier:
+  - **Allow list**. If the lowest version for each OS platform is a `[SupportedOSPlatform]` attribute, the API is considered to only be supported by the listed platforms and unsupported by all other platforms. The optional `[UnsupportedOSPlatform]` attributes for each platform can only have higher version of the minimum supported version, which denotes that the API is removed starting from the specified version.
+
+  ```csharp
+  // The API only supported on Windows 8.0 and later, not supported for all other.
+  // The API is removed/unsupported from version 10.0.19041.0.
+  [SupportedOSPlatform("windows8.0")]
+  [UnsupportedOSPlatform("windows10.0.19041.0")]
+  public void ApiSupportedFromWindows80SupportFromCertainVersion();
+  ```
+
+  - **Deny list**. If the lowest version for each OS platform is an `[UnsupportedOSPlatform]` attribute, then the API is considered to only be unsupported by the listed platforms and supported by all other platforms. The list could have `[SupportedOSPlatform]` attribute with the same platform but a higher version, which denotes that the API is supported starting from that version.
+  
+  ```csharp
```

Comment:
This code snippet and the one on lines 64-70 should be indented two more spaces to align with the list.